api/v1alpha3/types.go

```diff
@@ -23,22 +23,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// // ResourceSpec defines a generic spec that can used to define Azure resources.
-// TODO: ResourceSpec should be removed once concrete specs have been defined for all Azure resources in use.
-// type ResourceSpec interface{}
-
-// TODO: Write type tests
```

Comment:
**resolved**:  there are no testable functions in this file, only type definitions